scenario_tests/logical_replication/tests/test_logical_replication.py

```diff
@@ -14,63 +17,162 @@ def test_create_new_data(self):
             c.execute(
                 """
                 CREATE TABLE IF NOT EXISTS block (
-                    id serial NOT NULL,
-                    geom geometry(Polygon,4326),
+                    id serial NOT NULL primary key,
```

Comment:
Must add primary key so that update and delete can be replicated.

Reply:
Always a good idea to add a PK